ajax/libs/starability/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "starability",
+  "filename": "starability-minified/starability-all.min.css",
+  "version": "2.0.0",
+  "description": "Starability.css is an accessible rating system with cute animations on top. No JavaScript included.",
+  "author": "Anna Migas <amjmigas@gmail.com>",
+  "keywords": [
+    "rating",
+    "accessibility",
+    "css",
+    "sass",
+    "scss",
+    "animations"
+  ],
+  "homepage": "http://lunarlogic.github.io/starability/",
```

Comment:
@sashberd Sorry that I know that why need to change to `http`
Because this always redirect to `http://blog.lunarlogic.io/starability/`
Would you please help modify this?
Many thanks.